core/admin/mailu/models.py

```diff
@@ -255,20 +255,23 @@ def dns_autoconfig(self):
         """ return list of auto configuration records (RFC6186) """
         hostname = app.config['HOSTNAME']
         protocols = [
-            ('submission', 587),
-            ('imap', 143),
-            ('pop3', 110),
+            ('imap', 143, 20),
+            ('pop3', 110, 20),
+            ('submission', 587, 20),
         ]
         if app.config['TLS_FLAVOR'] != 'notls':
             protocols.extend([
-                ('imaps', 993),
-                ('pop3s', 995),
+                ('autodiscover', 443, 10),
+                ('submissions', 465, 20, 10),
```

Comment:
This is the problematic part that causes the error.  This should be 3 values. Maybe change it to
```
('submissions', 465, 10),
```

Reply:
Sorry... this is 0e9a1e6a